recipes-server/influxdb/files/influxd-systemd-start.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+
+/usr/bin/influxd &
+PID=$!
+echo $PID > /var/lib/influxdb/influxd.pid
+
+PROTOCOL="http"
+BIND_ADDRESS=$(influxd print-config --key-name http-bind-address)
+TLS_CERT=$(influxd print-config --key-name tls-cert | tr -d '"')
+TLS_KEY=$(influxd print-config --key-name tls-key | tr -d '"')
+if [ -n "${TLS_CERT}" ] && [ -n "${TLS_KEY}" ]; then
+  echo "TLS cert and key found -- using https"
+  PROTOCOL="https"
+fi
+HOST=${BIND_ADDRESS%:*}
+HOST=${HOST:-"localhost"}
+PORT=${BIND_ADDRESS##*:}
+
+set +e
+attempts=0
```

Comment:
Can we align to capitalics in naming as it's done earlier?
Please consider

Reply:
Done